doc/dune-files.rst

```diff
@@ -1988,40 +1964,43 @@ The loading of the plugin is done using the facilities generated by
 generate_sites_module (Since 2.8)
 ---------------------------------
 
-Dune proposes some facilities for dealing with :ref:`sites<sites>` in a program. The
-``generate_sites_module`` stanza will generate code for looking up the correct locations
-of the sites' directories and for loading plugins. It works after installation
-with or without the relocation mode, inside Dune rules, and when using Dune executables.
-For promotion, it works only if the generated modules are solely in the executable (or
-library statically linked) promoted; generated modules in plugins won't work.
+Dune proposes some facilities for dealing with :ref:`sites<sites>` in a program.
+The ``generate_sites_module`` stanza will generate code for looking up the
+correct locations of the sites' directories and for loading plugins. It works
+after installation with or without the relocation mode, inside Dune rules, and
+when using Dune executables. For promotion, it works only if the generated
+modules are solely in the executable (or library statically linked) promoted;
+generated modules in plugins won't work.
 
 .. code:: lisp
 
    (generate_sites_module
     (module <name>)
     <facilities>)
 
-The module's code is generated in the directory with the given name. The
-code is populated according to the requested facilities.
+The module's code is generated in the directory with the given name. The code is
+populated according to the requested facilities.
 
 
 The available ``<facilities>`` are:
 
-- ``sourceroot`` : adds in the generated module a value ``val sourceroot: string option``,
-  which contains the value of ``%{workspace_root}``, if the code have been built
-  locally. It could be used to keep the tool's configuration file locally when
-  executed with ``dune exec`` or after promotion. The value is ``None`` once it has been installed.
+- ``sourceroot`` : adds in the generated module a value ``val sourceroot: string
+  option``, which contains the value of ``%{workspace_root}``, if the code have
+  been built locally. It could be used to keep the tool's configuration file
+  locally when executed with ``dune exec`` or after promotion. The value is
+  ``None`` once it has been installed.
 
-- ``relocatable`` : adds in the generated module a value ``val relocatable: bool``,
-  which indicates if the binary has been installed in the relocatable mode
+- ``relocatable`` : adds in the generated module a value ``val relocatable:
```

Comment:
Since you haven’t used a colon before (that I can remember) in a list, consider:  `relocatable` adds a value `val relocatable: bool` in the generated module, which indicates…